### DISCUSSION TITLE
Release 3.7.0 - 🌦️ New Ping & Weather sensors, advanced radials, LHM update

### DISCUSSION THREAD

**Discussion by mathoudebine** [mathoudebine/turing-smart-screen-python#631] [Announcements]
## New Ping & Weather sensors
Thanks to @Psykotik, new sensors are available in this release:
- `PING` sensor to display ping delay between your computer and any other internal/external host/IP
- `WEATHER` sensors to display weather forecast (temperature/humidity/text forecast) from your location using OpenWeatherMap API

![image](https://github.com/user-attachments/assets/b1c905b6-67b7-4602-8fe0-1badcca4031f)

Check the new [ColoredFlat theme](https://github.com/mathoudebine/turing-smart-screen-python/blob/main/res/themes/ColoredFlat/theme.yaml) to get started!

## Advanced radial graphs
Thanks to @caerffili it is now possible to create intersecting radials:
![324036697-70a53157-98a8-4316-95ca-330fea0cbaf6](https://github.com/user-attachments/assets/0c9402f9-7ea1-4ce1-89a9-3d23bec242cb)

Check the [Advanced Radial theme](https://github.com/mathoudebine/turing-smart-screen-python/blob/main/res/themes/Advanced%20Radials%20Test/theme.yaml) example to get started!

## LibreHardwareMonitor 0.9.4 update
If you are using Windows, this release includes the new LibreHardwareMonitor release 0.9.4 to support latest hardware.
Check [the release note](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/releases/tag/v0.9.4) to see if your hardware is included in this release.

## New themes
ColoredFlat (5")
![preview-1](https://github.com/user-attachments/assets/3fc28d5e-597e-4029-8225-58557f32ebdb)

LandscapeModernDevice35 (3.5")
![preview](https://github.com/user-attachments/assets/7d6c57d7-3280-4cb8-b8ff-ef3302a275b6)

## Full changelog
* :arrow_up: Update pillow requirement from ~=10.4.0 to ~=11.0.0 by @dependabot in https://github.com/mathoudebine/turing-smart-screen-python/pull/586
* Incorrect log message when configuring manually in *nix environments. #582 by @nhomar in https://github.com/mathoudebine/turing-smart-screen-python/pull/583
* Added new theme by @napobear in https://github.com/mathoudebine/turing-smart-screen-python/pull/594
* Fix total_memory is NAN for AMD GPUs crashes thread by @Pepper-the-kobold in https://github.com/mathoudebine/turing-smart-screen-python/pull/573
* [Add] Ping and Weather custom sensor + ColoredFlat Theme by @Psykotik in https://github.com/mathoudebine/turing-smart-screen-python/pull/510
* Advanced radials by @caerffili in https://github.com/mathoudebine/turing-smart-screen-python/pull/500
* Replace relative path with absolute, to allow launch from another folder by @mathoudebine in https://github.com/mathoudebine/turing-smart-screen-python/pull/623
* Add LineGraph new properties: axis legend font and font size by @mathoudebine in https://github.com/mathoudebine/turing-smart-screen-python/pull/624
* :arrow_up: Update psutil requirement from ~=6.1.0 to ~=6.1.1 by @dependabot in https://github.com/mathoudebine/turing-smart-screen-python/pull/627
* Update LibreHardwareMonitor to 0.9.4 by @mathoudebine in https://github.com/mathoudebine/turing-smart-screen-python/pull/630

## New Contributors
* @nhomar made their first contribution in https://github.com/mathoudebine/turing-smart-screen-python/pull/583
* @napobear made their first contribution in https://github.com/mathoudebine/turing-smart-screen-python/pull/594
* @Pepper-the-kobold made their first contribution in https://github.com/mathoudebine/turing-smart-screen-python/pull/573
* @Psykotik made their first contribution in https://github.com/mathoudebine/turing-smart-screen-python/pull/510
* @caerffili made their first contribution in https://github.com/mathoudebine/turing-smart-screen-python/pull/500

**Full Changelog**: https://github.com/mathoudebine/turing-smart-screen-python/compare/3.6.0...3.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/mathoudebine/turing-smart-screen-python/releases/tag/3.7.0'>Release 3.7.0 - 🌦️ New Ping & Weather sensors, advanced radials, LHM update</a>.</em>

**Comment by DJRoby19** [comment]
Same problem since version 3.5.0 amd gpu fan is not detected also opened a bug but no response. This is the log
04/01/2025 21:30:08 [INFO] Detected AMD GPU(s)
04/01/2025 21:30:09 [WARNING] Your GPU FPS is not supported yet
04/01/2025 21:30:09 [WARNING] Your GPU Fan Speed ​​is not supported yet
